src/AppInitializer.tsx

```diff
@@ -114,7 +114,8 @@ class AppInitializer extends React.Component<Props, AppInitializerState> {
     })
   }
 
-  handleCloseLoginDialog = () => {
+  handleCloseLoginDialog = (event, reason) => {
+    if(reason && reason === "backdropClick") return
```

Comment:
Is this still necessary now that we have `backdrop='static'`?

Reply:
No it isn't. I'll remove this